src/stylesheets/components/_tile.scss

```diff
@@ -1,150 +1,57 @@
-// Basic tile
-// ----------------------
-
+//****************************
+//SDS TILE
 .sds-tile {
-  @include u-display('inline-block');
-  @include u-margin-x(2);
-}
-
-.sds-tile__content {
   @include u-display('flex');
-  @include u-flex('no-wrap');
-  @include u-flex('column');
-  @include u-text('center');
-  @include u-width('card-lg');
-  @include u-height('card');
-  text-decoration: none;
+  @include u-flex('align-center');
+  @include u-flex('justify-center');
 
-  &:hover {
-    text-decoration: none;
-  }
-  @media only screen and (min-width: 600px) {
-    .sds-tile__icon {
-      @include u-margin-x('auto');
-      @include u-margin-bottom('105');
-      text-decoration: none !important;
-      svg.fa-stack-1x{
-        font-size: 2em;
-      }
-    }
+  .sds-tile__body {
+    @include u-display('block');
+    @include u-padding-left('2');
 
-    .sds-tile__title {
-      @include u-font('sans', 'sm');
+    header {
+      @include u-font('sans', 'md');
       @include u-text('bold');
-      @include u-text('black');
-    }
 
-    .sds-tile__details {
-      @include u-font('sans', 'xs');
-      @include u-text('black');
     }
-  }
-  @media only screen and (max-width: 600px) {
-      @include u-display('flex');
-      @include u-flex('column');
-      width: 150px;
+    .sds-list {
+      @include u-margin-top(0);
+      @include u-padding-left(0);
+      list-style-type: none;
 
-    .sds-tile__icon {
-      align-self: center;
-      margin-top: 10px;
-      width: 60px;
-      text-decoration: none !important;
-
-      svg {
-        height: 60px !important;
-        width: 60px !important;
+      li {
+        @include u-padding-bottom(0);
       }
     }
-
-    .sds-tile__title {
-      @include u-font('sans', 'sm');
-      @include u-text('bold');
-      @include u-text('black');
-    }
-
-    .sds-tile__details {
-      @include u-font('sans', 'xs');
-      @include u-text('black');
-    }
   }
 }
-// Outline tile
-// ----------------------
 
-.sds-tile__content.sds-tile--outline {
-  @include u-border(2px, 'white');
-  @include u-padding(2);
-  @include u-margin-top(4);
-  background: rgba(240, 245, 249, 0.8);
-  text-decoration: none !important;
-  border-radius: 2rem;
-  @media only screen and (min-width: 600px) {
-    height: 150px;
-    width: 195px;
+//****************************
+//SDS TILE OUTLINE
+.sds-tile--outline {
+  @include u-border('2px', 'white');
+  @include u-padding(4);
+  @include u-bg('accent-cool-lighter');
+  @include u-width('full');
+  border-radius: 20px;
```

Comment:
Please use border radius and opacity tokens

Reply:
there isn't a border radius token large enough in uswds to use here